src/pages/api/productDateInformation.ts

```diff
@@ -0,0 +1,165 @@
+import { NextApiResponse } from 'next';
+import * as yup from 'yup';
+import moment from 'moment';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { PRODUCT_DATE_INFORMATION } from '../../constants';
+import { ErrorInfo, NextApiRequestWithSession } from '../../interfaces';
+import { redirectTo, redirectToError } from './apiUtils';
+
+export interface ProductDateAttribute {
```

Comment:
Conversely, might be best to not call this an attribute and call it information, if we rename the session string to an attribute.